iohomework/src/main/java/Main.java

```diff
@@ -0,0 +1,30 @@
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+        File f = new File ("product-sales.csv");
+        if (!f.exists()) {
+            System.out.println("File " + f.getName() + " does not exist.");
+        }
+
+        FileReader fr = new FileReader(f);
+        List<Sales> sales = new SalesReader(fr).getSales();
+        /*System.out.println("\nSales: ");
```

Comment:
Never commit commented out code. Use a local branch if you wish to store it for later use.

Reply:
ok...i'll remove